docs/sample_queries/gnomad/gnomad.ipynb

```diff
@@ -6824,6 +6824,150 @@
+            "9ea1076c95ce44f2ae1d5d6439b1a500",
+            "5b9b317ab4084e32ba003b247da41259",
+            "033fcdc3f7b2430484c2df7d9e489762",
+            "cd26e7308f3b46718205df89eb3078d6",
+            "08e1fbb08edc461d8ba2d150632a4ac3",
+            "680ce42324474b188af071a6a9411a14",
+            "ac25fd0897b94e0db9f754c0978eae3f",
+            "3aba328911b44a7386c52eba91365d2f",
+            "3b2601226d9d4738ba5941da4488be7e",
+            "bdef67ea6f8643ce8f8bb276c9f974f8",
+            "5898c56ac6de43a7a0cdea178d49ac7a",
+            "6fe7a972eb24464c86c6492570af71ba",
+            "95f51241a37a491aa57f1320880bc73a",
+            "a3e46674df32443e832f06735dfad04d",
+            "7f3c686a90e4463c8ec80b99da814229",
+            "c56d8cb88f1d4ecbb8500cfaa7b4f5c1",
+            "e8444aeb52314482b52bc4c12a0baf7f",
+            "02d12605a2c442738b6033f0ee0670fc",
+            "db6d934138bc4f749253e4fe033c5f11",
+            "655d8f41cbfd427d9a822c5c2e84a766",
+            "2ebb60be78d9452fa8516d6a6b7a8a46",
+            "b9a5e90d88d84ead945e30d18b763ce2",
+            "3f7c89cbdab34cca84d36f52f9a1f7d8",
+            "99ddecfe558044bbb004813a32bbca5d",
+            "0167e67ae7cd4adcb85172f261314719",
+            "33134e953ed94174b805b9994a8f7f21",
+            "ec19667657ed446b87a2125f25c6db65",
+            "8d399dd46af34e5d90026f2e68f1437a",
+            "e8fdcf196f494956b87e48c979859bbf",
+            "841ecd36daab452bb1f6111446c40978",
+            "b74ff413fe9c424184ffd0061e6c5275",
+            "5497c5b104b0413995225595f4e32aca",
+            "ab17626978ed4d2b8eabb3f3164ea446",
+            "9ad3cf1d5d30453fbaf6dad79e84ca90",
+            "b3e4db7b285b43539ddc1f8107268672",
+            "8ab0d5c923eb41b2a4bdcbf21e1a7abc",
+            "35b1d58488c54cecb462292bf83a284a",
+            "ead19d874a3d45e6830533c790e1f226",
+            "69e0c48a2d00477c949fa070a7f4be27",
+            "17647b9501354aadadf1f93d68cb1e1a",
+            "2aaed994b4434552a4aa4de7d9e089de",
+            "b36b1709c7764ebc92631c7bc6e21190",
+            "cc0cb4296ec24ae3adf1f23fe0b580c6",
+            "70b9a6f889064af798b71db603623317",
+            "223f2f02fe3f488c9c9dec61a08fb0a0",
+            "877b5aac9e7647d98f40731ea8dcf813",
+            "d9d6bd8b9f9145c0a1179a92f71fb2fb",
+            "aa6a83929d9743a3997b2f64510210e0",
+            "72ab6b3c9937404cb15476f5d11b40f9",
+            "7f9958af58594d4fbdc78e6e9d9eeea6",
+            "f85fa1f9b9ac492faef418a42cce68d2",
+            "6e8b181369a24b319f9f747ee419009e",
+            "58fa9b37fa0048c297f7e8ad20d1a26e",
+            "752d48dee5ac4092bf1e3e0100960772",
+            "fd6f1303ad6a454cabac426f14c78f16",
+            "4203f98b3a4b4723b398310a4691cb8c",
+            "a54bb74bb77244b2a74ad456b0a5f12f",
+            "03805018509243698bc8ac65990ce734"
+          ]
+        }
+      },
+      "source": [
+        "query_template = \"\"\"\n",
+        "SELECT count(1) AS num_mutations\n",
```

Comment:
Please update the query cell as follows:
```
bucket_size = 10000
query_template = """
SELECT CAST(FLOOR(DIV(start_position, {BUCKET})) AS INT64) AS start_pos_bucket ,
       count(1) AS num_snv
FROM 
    (
    SELECT DISTINCT 
          start_position,
          alternate_bases.alt AS alt,
          {VAR_TYPE_COL} AS variant_type,
    FROM `bigquery-public-data.gnomAD.{GNOMAD_VER}__{CHROM}` AS main_table,
        main_table.alternate_bases AS alternate_bases
    WHERE variant_type = 'snv'
    )
GROUP BY 1
ORDER BY 1
"""
query = query_template.format(GNOMAD_VER=gnomad_version_chr,
                              CHROM=chromosome_chr,
                              VAR_TYPE_COL=variant_type_col,
                              BUCKET=bucket_size)

snv_dist = run_query(query)
snv_dist.head()

```

Reply:
Then add a second cell to visualize the result:
```
import matplotlib.pyplot as plt

plt.figure(dpi=150)
plt.bar(snv_dist.start_pos_bucket, snv_dist.num_snv)
plt.xlabel("Bucket number of start_pos")
plt.ylabel("No of SNVs in each bucket")
plt.title("Distribution of SNVs on {} for buckets of {} base pairs".format(chromosome_chr, bucket_size))
plt.show()
```

Reply:
Hi Saman, thank you so much for these suggestions. I have updated my PR, have a good day!